libs/features/links.js

```diff
@@ -13,16 +13,13 @@ const getNoFollowLinks = (() => {
 })();
 
 export default async function init(path, area = document) {
-  if (!path) return null;
   const data = await getNoFollowLinks(path);
+  if (!data) return null;
   const links = area.querySelectorAll('a:not([href^="/"])');
-  return [...links].map((link) => {
-    data.forEach((site) => {
-      if (link.href.startsWith(site.domain)) {
-        link.setAttribute('rel', 'nofollow noopener noreferrer');
-        link.setAttribute('target', '_blank');
-      }
-    });
-    return link;
+  [...links].forEach((link) => {
+    data.filter((s) => link.href.startsWith(s.domain)).map((s) => {
```

Comment:
nit: new line for `.map`.  Also should probably use `forEach` since the map output isn't used anywhere?

Reply:
an excellent nit